packages/codegen/package.json

```diff
@@ -16,13 +16,13 @@
   "scripts": {
     "tsc": "tsc",
     "lint": "eslint",
-    "copy-hygen": "cp -r src/codegen/.hygen dist/src/codegen/",
+    "copy-hygen": "rm -rf dist/src/codegen/.hygen && cp -r src/codegen/.hygen dist/src/codegen/",
     "build": "npm run copy-hygen",
     "build:create-lib": "tsc --project src/create-lib/tsconfig.json",
     "publish:create-lib": "npm publish --cwd ./src/create-lib/",
     "start": "tsx src/cli.ts",
```

Comment:
having multiple different places where we output generated code from tests is needlessly confusing and causes issues with typescript and git